test-it/karma/publicapi/Double.Test.js

```diff
@@ -56,12 +56,6 @@ describe('The Vireo EggShell Double api can', function () {
         expect(readDouble(path)).toMatchIEEE754Number(initialValue);
     };
 
-    // Expected coerced values of Doubles in Vireo should have the same behavior as assignment to typed arrays
-    // This is so a user assigning a double to a typedarray using getTypedArray will see the same effect as writing to a double using writeDouble
-    var expectedCoercedValue = function (TypedArrayConstructor, value) {
```

Comment:
Can you add a comment to [readTypedArray](https://github.com/ni/VireoSDK/blob/c8eb11d197b09f7ed2100bdb01cd48cd8f2fa003/source/io/module_eggShell.js#L472) that describes the change in assumption. Something like:


```js
// Note: If you are directly writing JavaScript numbers (floating point doubles) to 
// integer TypedArray views then be aware that integer TypedArray views will truncate
// overflowing values as specified in TypedArray conversion operations:
// https://tc39.es/ecma262/#table-the-typedarray-constructors
// Vireo requires that overflowing JavaScript numbers written to Vireo integers
// are saturated instead of truncated. Make sure to perform Vireo's float-to-int 
// conversion algorithm before directly writing to integer TypeArray views.
Module.eggShell.readTypedArray = publicAPI.eggShell.readTypedArray = function (valueRef) { ... };
```

And to [writeTypedArray](https://github.com/ni/VireoSDK/blob/c8eb11d197b09f7ed2100bdb01cd48cd8f2fa003/source/io/module_eggShell.js#L491):

```js
// Note: When creating integer TypedArrays views from JavaScript numbers to write to
// Vireo using writeTypedArray be aware that integer TypedArray views will truncate
// overflowing values as specified in TypedArray conversion operations:
// https://tc39.es/ecma262/#table-the-typedarray-constructors
// Make sure to perform Vireo's float-to-int conversion when creating integer TypedArray
// views from JavaScript numbers to write to Vireo memory.
Module.eggShell.writeTypedArray = publicAPI.eggShell.writeTypedArray = function (valueRef, typedArrayValue) {
```

Reply:
Added above comments.